src/js/selection/SelectionManager.js

```diff
@@ -91,7 +93,29 @@
     }
   };
 
-  ns.SelectionManager.prototype.paste = function() {
+  // Iterate through pixels to get the largest x and y values of a selection.
+  ns.SelectionManager.prototype._getBoundaryEdges = function(pixels) {
+    var xCoordinate = 0;
+    var yCoordinate = 0;
```

Comment:
Not relevant in this particular case but it's possible for a selection to (at least temporarily) exist entirely above or to the left of the canvas.  It might be more precise here to initialize these to `-Infinity`.